src/components/Categories/MainCategoryTile.vue

```diff
@@ -1,59 +1,56 @@
 <template>
 	<div v-if="tileSize === 'large'" class="tw-mb-6">
 		<img class="tw-rounded tw-w-full"
-			:src="image"
+			:src="categoryImage"
```

Comment:
You'll need to use a responsive image here to make sure that on retina screens the category retina image is used, but on non retina screens the non retina image is used. This makes the page load faster since the smaller screens don't have to load the large images. You should also lazy load these images to increase performance, this will make it so the small tile images dont have to be fetched if the small tiles are not in view. 

You can take a look at the `kv-responsive-image` component that handles this functionality